ChangeLog.d/driver-only-hashes.txt

```diff
@@ -8,12 +8,16 @@ Features
      are only provided by PSA drivers. In these configurations, you need to
      call `psa_crypto_init()` before you call any function from those
      modules; this is not required in configurations where the built-in
-     implementation is still available. Note that some crypto modules and
-     features still depend on the built-in implementation of hashes:
-     MBEDTLS_HKDF_C (but the PSA HKDF function do not depend on it),
-     MBEDTLS_ENTROPY_C, MBEDTLS_HMAC_DRBG_C and MBEDTLS_ECDSA_DETERMINISTIC.
-     In particular, for now, compiling without built-in hashes requires use
-     of MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG.
+     implementation is still available. Note that for modules that use MD
+     (RSA, PKCS5, PKCS12, EC J-PAKE) in builds that have MBEDTLS_MD_C enabled,
+     all hashes used with those modules need to be built-in, as drivers are only
+     used when MBEDTLS_MD_C is disabled; configurations where some hashes are
+     available as built-ins, and some only from drivers, are currently not
```

Comment:
This seems overly restrictive. What we don't support is a mix of hashes that are only available via the legacy interface but not PSA, and others that are only available as PSA drivers. If all hashes are available through the PSA interface, it's fine, regardless of whether they're provided via drivers or not.